regression/goto-instrument-unwind/unwind-chain.sh

```diff
@@ -6,22 +6,18 @@ goto_instrument=$src/goto-instrument/goto-instrument
 cbmc=$src/cbmc/cbmc
 
 function usage() {
-  echo "Usage: chain unwind-num test_file.c"
+  echo "Usage: chain <options> <testfile>"
   exit 1
 }
 
-echo "Usage: process test_file.c to goto program"
-
-if [ $# -eq 2 ]
-then
-  unwind_num=$1
-  name=`echo $2 | cut -d. -f1`
-  echo $unwind_num
-  echo $name
-else
+if [ $# -ne 2 ]; then
   usage
+  exit 1
 fi
 
+name=$(echo $2 | cut -d. -f1)
+
 $goto_cc -o $name.gb $name.c
-$goto_instrument  $name.gb ${name}-unwound.gb --unwind $unwind_num
```

Comment:
$unwind_num is actually undefined now. With the `shift` above use:
$goto_instrument $name.gb ${name}-unwound.gb $@